teamcity.sh

```diff
@@ -2,5 +2,34 @@
 
 set -e
 
-npm install
-npm run build
+setupNvm() {
+  export NVM_DIR="$HOME/.nvm"
+  [[ -s "$NVM_DIR/nvm.sh" ]] && . "$NVM_DIR/nvm.sh"  # This loads nvm
+
+  nvm install
+  nvm use
+}
+
+
+buildJs() {
+  echo "##teamcity[compilationStarted compiler='js']"
+
+  setupNvm
+
+  # clear old packages first
+  rm -rf node_modules
+
+  npm install
+  npm run build
+
+  echo "##teamcity[compilationFinished compiler='js']"
+}
+
+buildSbt() {
+  echo "##teamcity[compilationStarted compiler='sbt']"
+  sbt clean test scripts/compile riffRaffUpload
```

Comment:
Should `scripts/compile` be `compile`?

Reply:
Sorry for reviewing this in an awkward way!

Reply:
Aw, I thought I corrected this before you spotted it! Yes it should!

Reply:
This should be done now. At least I've force pushed it, but GitHub is taking a little while to see it.